### PR TITLE
Chinese translation for swipe tutorial.

### PR DESCRIPTION
**Pull request by gebitang**
## Proposed changes

Just add Chinese translation for swipe tutorial.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: docs/cn/writing-running-appium/tutorial/swipe/swipe-troubleshoot-guide.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+### iOS: 'mobile:scroll', 'mobile:swipe' 滑动(swipe)
+
+#### 滑动没有开始：
+
+1. 检查方向，注意'scroll'和'swipe'方法的方向参数不同。
+2. 都失败的话，尝试使用简单元素滑动方式。
+
+#### 找不到查找的元素:
+
+1. 是否需要使用请准swipe而不是scroll方法
```
Comment (lihuazhang) [inline docs/cn/writing-running-appium/tutorial/swipe/swipe-troubleshoot-guide.md]: 是否需要使用请准swipe而不是scroll方法？看不懂

File: docs/cn/writing-running-appium/tutorial/swipe/ios-picker-wheels-set-value.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## iOS pickerWheels: 快速方式 '.setValue()' 示例
+
+不幸的是，XCTest不能总是很好地与轮式选择器控件(PickerWheel)进行交互。有时，setValue()方法调用可能没有任何效果。
```
Comment (lihuazhang) [inline docs/cn/writing-running-appium/tutorial/swipe/ios-picker-wheels-set-value.md:3]: 不能总是===》总是不能 是不是好一点。你读一下看看？

**Review by lihuazhang — CHANGES_REQUESTED** [review]
翻译的挺不错的。有几个点你看看改一下。

File: docs/cn/writing-running-appium/tutorial/swipe/swipe-troubleshoot-guide.md (hunk 1 above)
Comment (gebitang) [reply docs/cn/writing-running-appium/tutorial/swipe/swipe-troubleshoot-guide.md]: If you need precise swipe prefer scroll method. 

修改为“尝试使用精准swipe而不是scroll方法”

请准 是个 拼写错误

File: docs/cn/writing-running-appium/tutorial/swipe/ios-picker-wheels-set-value.md (hunk 2 above)
Comment (gebitang) [reply docs/cn/writing-running-appium/tutorial/swipe/ios-picker-wheels-set-value.md:3]: 不能总是：大部分情况OK，少部分情况不OK

总是不能：大部分情况都不OK

**Pull request merged**